autoPyTorch/pipeline/components/training/losses.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Dict, Optional, Type
+
+from torch.nn.modules.loss import (
+    CrossEntropyLoss,
+    BCEWithLogitsLoss,
+    MSELoss,
+    L1Loss
+)
+from torch.nn.modules.loss import _Loss as Loss
+
+
+losses = dict(classification=dict(
+    CrossEntropyLoss=dict(
+        module=CrossEntropyLoss, supported_output_type='multi-class'),
```

Comment:
Ideally, we should have a single mapping like https://github.com/automl/auto-sklearn/blob/7a3f3a517821da7f00d92991960dd474a22c223e/autosklearn/constants.py#L16

which we can use here. I think @bastiscode  was creating something like this... maybe you can use what he is using?

Reply:
I agree, he is on vacation this week though

Reply:
So, by single mapping you mean the same for loss and metrics? I didnt quite understand what you mean, could you explain further on where this mapping is to be used?

Reply:
Of course. We all have to speak the same language. 

If I ask in my part of the code for 'binary_classification' and you in your code, you ask for 'BINARYCLASSIFICATION' every time you/me have the need to query this information, it becomes tedious. 

Can you please create a file named constants.py like the one above, where you use an enumerate so we standardized the question:
```
If BINARY_CLASSIFICATION:
    # Do Something
```
Where BINARY_CLASSIFICATION will be a enum object in this file that you create